molecule/test/resources/playbooks/docker/Dockerfile.j2

```diff
@@ -10,7 +10,7 @@ ARG testarg=fail
 ENV envarg=$testarg
 
 RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y python sudo bash ca-certificates iproute2 python-apt aptitude && apt-get clean; \
-    elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python sudo python-devel python*-dnf bash iproute && dnf clean all; \
+    elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python2 python36 sudo python2-devel python36-devel python*-dnf bash iproute && dnf clean all; \
```

Comment:
While I can understand the need workaround and force use of centos 7 while we add support for the new one, I do not think we should touch this file in the same change, especially as installing multiple pythons is not to be desired.

Reply:
Ah, that was supposed to be removed (before, I was trying to support CentOS 8 and it has no `python` package, only `python2` and `python36` etc.).